theme/less/_navigation.less

```diff
@@ -1,25 +1,103 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  height:50px;
+  display: flex;
+  padding: 0;
+  border-color: #EAEAEA;
+  border-radius: 5px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+#main-navigation > ul li
+{
+  padding: 15px;
```

Comment:
We might need to use this padding on the anchor tag, because currently the link is only working when you hover on the text, but not on the menu item, same applies to the padding of subitem li element, which might also need to be moved into their anchor tag accordingly.

Reply:
Resolved